projects/httpcomponents-client/FileBodyWriteToFuzzer.java

```diff
@@ -0,0 +1,54 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueLow;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.apache.hc.client5.http.entity.mime.FileBody;
+import org.apache.hc.core5.http.ContentType;
+
+public class FileBodyWriteToFuzzer {
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+        // Create objects from fuzzer input
+        final ContentType contentType = data.pickValue(contentTypes);
+        final String filename = data.consumeString(255);
+        final String fileContent = data.consumeRemainingAsString();
+
+        // Create needed objects
+        final File tempFile;
+        try {
+            tempFile = File.createTempFile("FileBody", "bin");
+            tempFile.deleteOnExit();
```

Comment:
It may be better to use `tempFile.delete()` as otherwise the temp files will only be deleted once the jvm stop, which can result in a lot of files if the fuzzer is running for a long time

Reply:
You are right.
I should put `tempFile.delete()` in `fuzzerTearDown()`.